toolkit/templates/instruction-list.html

```diff
@@ -64,6 +64,24 @@ <h2 class="instruction-list-item-body">{{ item.body }}</h2>
             {% endwith %}
           {% endfor %}
         {% endif %}
+
+        {% if item.content %}
+          {% for content in item.content %}
```

Comment:
The naming here is a bit yuck. Is there any possibility of changing at this point? Having content as a _list_ containing content as a _dict_ means you never really know what _content_ actually means!